### DISCUSSION TITLE
Remote control

### DISCUSSION THREAD

**Discussion by tuskiie** [helix-editor/helix#6805] [Ideas] [answered]
Any thoughts on allowing helix to be controlled remotely like using [neovim-remote](https://github.com/mhinz/neovim-remote)?

**Comment by michaeljones** [comment, marked as answer]
I have a patch that I've been applying that I use in order to be able to open more files into the running helix instance from my command line, which is what I used neovim-remote for. 

Only works on unix as that is all I need and only supports the 'open' command but probably isn't too hard to expand. It is hacky and best paired with a shell script to help manage the open socket that is used for communication. The details are here: https://github.com/helix-editor/helix/discussions/5107#discussioncomment-4549555

**Reply by tuskiie** [reply]
Not sure why your discussion didn't come up when I searched. This looks good I'll try it out, thank you!
